Задание_№9/Задание_№9/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+
+namespace Задание__9
+{
+    class Program
+    {
+        static void Main()
+        {
+            int[] mas = new int[10];
+            int a, d;
+            Console.WriteLine("Введите первое значение прогресси");
+            a = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Введите разность прогресси");
+            d = Convert.ToInt32(Console.ReadLine());
+            nomber(ref mas, a, d);
```

Comment:
Массив предается в метод по ссылке. Использовать `ref` нет смысла.
```suggestion
            FillArray(mas, a, d);
```